src/test/java/htsjdk/samtools/cram/build/CompressionHeaderFactoryTest.java

```diff
@@ -3,110 +3,79 @@
 import htsjdk.HtsjdkTest;
 import htsjdk.samtools.ValidationStringency;
 import htsjdk.samtools.cram.encoding.readfeatures.Substitution;
-import htsjdk.samtools.cram.structure.CompressionHeader;
-import htsjdk.samtools.cram.structure.CramCompressionRecord;
-import htsjdk.samtools.cram.structure.EncodingID;
-import htsjdk.samtools.cram.structure.DataSeries;
-import htsjdk.samtools.cram.structure.ReadTag;
-import htsjdk.samtools.cram.structure.SubstitutionMatrix;
+import htsjdk.samtools.cram.structure.*;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
 import java.util.ArrayList;
 import java.util.Collections;
-import java.util.List;
 
 /**
  * Created by vadim on 07/01/2016.
  */
 public class CompressionHeaderFactoryTest extends HtsjdkTest {
-    @Test
-    public void testAllEncodingsPresent() {
```

Comment:
Can you explain why you removed this test case?

Reply:
Its replaced by a series of tests in `CompressionHeaderEncodingMapTest`, i.e., `testAllHTSJDKWriteEncodingsRoundTripThroughStream` and `testAllCRAMEncodingsRoundTripThroughStream`.